### PR TITLE
Change log: fix broken page anchor url encodings

### PR DESCRIPTION
**Pull request by yangjunmyfm192085**
…e-scheduler-and-cloud-controller-manager]

Signed-off-by: JunYang <yang.jun22@zte.com.cn>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Change log:Fix  hyperlink of description [kube-controller-manager-kube-scheduler-and-cloud-controller-manager] 
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

**Comment by tpepper** [comment]
This is funky.  Looking at where these came from so it can be fixed upstream of the generated .md file.

@yangjunmyfm192085 there are a couple other instances of `/&#35;` in the file.  Can you fix those in this PR also?

**Comment by yangjunmyfm192085** [comment]
will be mentioned next time pr

> This is funky. Looking at where these came from so it can be fixed upstream of the generated .md file.
> 
> @yangjunmyfm192085 there are a couple other instances of `/&#35;` in the file. Can you fix those in this PR also?
Thanks for attention。
I noticed this now，and found other errors。
I'm still reading the rest changelogs。I‘ll try fix them all in next pr。
Let's merge this pr first

**Comment by tpepper** [comment]
> > I'm still reading the rest changelogs。I‘ll try fix them all in next pr。
> > Let's merge this pr first

There's just a small bit of additional:
```
$ git grep "/&#35;" | wc -l
       7
$ git grep -l "/&#35;"
CHANGELOG/CHANGELOG-1.18.md
CHANGELOG/CHANGELOG-1.19.md
CHANGELOG/CHANGELOG-1.20.md
```

There's little urgency on this individual change and it would be more efficient from a review perspective to get the situation fully addressed by fixing all 7 instances of the error in the 3 files via one commit in one PR.  Separately we've got eyes looking at the generator to make sure no new ones are added.

Can you please make that minor amendment to this PR?

**Comment by tpepper** [comment]
Also @yangjunmyfm192085 if you're new the workflow of amending existing git commits and GitHub PRs, poke me on Slack (same userid as here) and I'm happy to help walk you through it.  It's super quick and easy!

**Comment by wilsonehusin** [comment]
+1 to Tim's comment, I'd like to get it resolved at once while I investigate for this not to happen again in new ones being generated.

**Comment by yangjunmyfm192085** [comment]
ok.
Thank you for reminding,I have corrected all the incorrect use of  “/&#35;”(Not all need to be modified).

**Comment by yangjunmyfm192085** [comment]
Thank you very much. I really like kubernetes and I have just started researching it.
I really hope get help in the future work.

**Comment by yangjunmyfm192085** [comment]
> Also @yangjunmyfm192085 if you're new the workflow of amending existing git commits and GitHub PRs, poke me on Slack (same userid as here) and I'm happy to help walk you through it. It's super quick and easy!



> Also @yangjunmyfm192085 if you're new the workflow of amending existing git commits and GitHub PRs, poke me on Slack (same userid as here) and I'm happy to help walk you through it. It's super quick and easy!

Thank you very much. I'm interested in kubernetes and learning it recently,I hope to get some help in the future works,thanks again.

**Comment by wilsonehusin** [comment]
thanks @yangjunmyfm192085!

/lgtm

**Comment by tpepper** [comment]
/retitle Change log: fix broken page anchor url encodings

/lgtm

It turns our there are another 17 or so additional instances of `https:/f.q.d.n/file.md&#35;anchor` and `https:/f.q.d.n/string&#35;anchor` which also need fixed up.  And a few cases of ` &#35; ` in text (not urls) which does render correctly.

```bash
git grep -l -e "&#35;"
CHANGELOG/CHANGELOG-1.17.md
CHANGELOG/CHANGELOG-1.18.md
CHANGELOG/CHANGELOG-1.19.md
CHANGELOG/CHANGELOG-1.20.md
```
Putting the leading '/' in the regex was hiding a few. :\  

If @yangjunmyfm192085 can fix those in this same PR too that would be awesome, else I've left a note of the oddity in https://github.com/kubernetes/release/issues/1873.

**Comment by wilsonehusin** [comment]
hi @yangjunmyfm192085, would you be able to incorporate the changes mentioned by Tim above as well in this PR?

**Comment by yangjunmyfm192085** [comment]
> hi @yangjunmyfm192085, would you be able to incorporate the changes mentioned by Tim above as well in this PR?

I'm doing it,but there are some problems.
For example:
- Bug fixes: Make sure we include latest packages node &#35;351 (@caseydavenport)
I can not find issue 351 or pr 351 about caseydavenport,Cannot add hyperlink.
I need only remove extra  "&#35;"?
But remaining  "&#35;" would not be displayed on the page.

**Comment by wilsonehusin** [comment]
@yangjunmyfm192085 heh I don't know what `#351` is in reference to either. if you wrap it back ticks I'll be okay with it and move on. something like:
```
Make sure we include latest packages node `#351`
```

**Comment by yangjunmyfm192085** [comment]
Hi，@wilsonehusin @tpepper ，I have checked all " \&#35"".
Could you review it again? Thank you very much.
Any questions,I'll deal with it at once.

File: CHANGELOG/CHANGELOG-1.19.md [hunk 1]
```diff
@@ -1177,7 +1177,7 @@ The new storage capacity tracking alpha feature is known to be affected by a lim
...
-- Added pod version skew strategy for seccomp profile to synchronize the deprecated annotations with the new API Server fields. Please see the corresponding section [in the KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190717-seccomp-ga.md&#35;version-skew-strategy) for more detailed explanations. ([#91408](https://github.com/kubernetes/kubernetes/pull/91408), [@saschagrunert](https://github.com/saschagrunert)) [SIG Apps, Auth, CLI and Node]
+- Added pod version skew strategy for seccomp profile to synchronize the deprecated annotations with the new API Server fields. Please see the corresponding section [in the KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/135-seccomp/README.md&#35;version-skew-strategy) for more detailed explanations. ([#91408](https://github.com/kubernetes/kubernetes/pull/91408), [@saschagrunert](https://github.com/saschagrunert)) [SIG Apps, Auth, CLI and Node]
```
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.19.md:1180]: the new path is correct, but I think the `&#35;` still needs to be fixed here 😄

File: CHANGELOG/CHANGELOG-1.19.md [hunk 2]
```diff
@@ -1309,15 +1309,15 @@ The new storage capacity tracking alpha feature is known to be affected by a lim
...
-    cachingMode: None  &#35; ReadOnly host caching is not available for premium SSDs with maxShares>1
-    maxShares: 2 ([#89328](https://github.com/kubernetes/kubernetes/pull/89328), [@andyzhangx](https://github.com/andyzhangx)) [SIG Cloud Provider and Storage]
+  - kind: StorageClass
+  - apiVersion: storage.k8s.io/v1
+  - metadata:
+    - name: shared-disk
+  - provisioner: kubernetes.io/azure-disk
+  - parameters:
+    - skuname: Premium_LRS  #Currently only available with premium SSDs.
+    - cachingMode: None #ReadOnly host caching is not available for premium SSDs with maxShares>1
```
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.19.md:1319]: missing space between `#` and the first character, this should be
```
    - skuname: Premium_LRS  # Currently only available with premium SSDs.
    - cachingMode: None # ReadOnly host caching is not available for premium SSDs with maxShares>1
```

File: CHANGELOG/CHANGELOG-1.19.md [hunk 3]
```diff
@@ -3404,21 +3404,21 @@ filename | sha512 hash
...
-    skuname: Premium_LRS  &#35; Currently only available with premium SSDs.
-    cachingMode: None  &#35; ReadOnly host caching is not available for premium SSDs with maxShares>1
+  - kind: StorageClass
+  - apiVersion: storage.k8s.io/v1
+  - metadata:
+    - name: shared-disk
+  - provisioner: kubernetes.io/azure-disk
+  - parameters:
+    - skuname: Premium_LRS  #Currently only available with premium SSDs.
+    - cachingMode: None  #ReadOnly host caching is not available for premium SSDs with maxShares>1
```
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.19.md:3421]: same as above, need space after `#`

File: CHANGELOG/CHANGELOG-1.20.md [hunk 4]
```diff
@@ -2158,7 +2158,7 @@ filename | sha512 hash
...
-- Added pod version skew strategy for seccomp profile to synchronize the deprecated annotations with the new API Server fields. Please see the corresponding section [in the KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190717-seccomp-ga.md&#35;version-skew-strategy) for more detailed explanations. ([#91408](https://github.com/kubernetes/kubernetes/pull/91408), [@saschagrunert](https://github.com/saschagrunert)) [SIG Apps, Auth, CLI and Node]
+- Added pod version skew strategy for seccomp profile to synchronize the deprecated annotations with the new API Server fields. Please see the corresponding section [in the KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/135-seccomp/README.md&#35;version-skew-strategy) for more detailed explanations. ([#91408](https://github.com/kubernetes/kubernetes/pull/91408), [@saschagrunert](https://github.com/saschagrunert)) [SIG Apps, Auth, CLI and Node]
```
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:2161]: same as above, need to convert `&#35;` --> `#`

**Review by wilsonehusin — CHANGES_REQUESTED** [review]
We're getting close, thanks for looking into this and let's get this done! :)

**Comment by yangjunmyfm192085** [comment]
@wilsonehusin
Sorry,my mistake,already corrected.
thanks.

**Review by wilsonehusin — APPROVED** [review]
Thanks for contributing!

/lgtm
/approve

**Pull request merged**